modules/ROOT/pages/anypoint-platform-cli-commands.adoc

```diff
@@ -1175,6 +1175,9 @@ When the `--include-dot-files` option is used, the command uploads hidden files
 
 Besides the `--include-dot-files`, this command takes the default `--help`, `-f`/`--fields` and `-o`/`--output` options.
 
+[NOTE]
+This command only uploads a file to an existing project in your Design Center. It does not create a new project if it does not exist. To create a new project, use the `designcenter project create` command.
```

Comment:
Actually, it does not only upload a file. It uploads all files included in the directory specified in the `projDir` parameter. Can we clarify this to prevent further misunderstandings?